PatientRegistrationApp/src/customerSettings.js

```diff
@@ -1,16 +1,16 @@
 export default {
     householdDefinition: 'A household is composed of any people who occupy the same given housing unit. If you are planning to register your household, all members must currently be residing in the same address. If members of a household are residing in other housing units, it is necessary to register any given members individually. Max household registration of 20 registrants.',
-    
+
     contactInfoDisclosure: 'We ask that you please provide your phone number and/or e-mail address.  This contact information will provide us with the ability to send you important information regarding your vaccination registration, follow-up appointment, and vaccination summary.',
     //contactInfoDisclosure: 'We ask that you please provide your phone number and/or e-mail address. This will make it possible for us to send you information about your vaccination registration, follow-up appointment, and vaccination summary.',
-    
+
     contactInfoConsequence: 'By withholding your e-mail and/or phone number, it will not be possible to provide you with the important vaccination information.',
     //contactInfoConsequence: 'This information will allow us to follow up on any details that need to be handled quickly.',
-    
+
     contactInfoAcknowledgement: 'You understand that we will use the phone number and e-mail that you provide to contact you with vaccination registration and follow-up information.',
 
     welcome: 'Welcome!',
-    introStatement: 'You are about to register as a patient for the COVID-19 vaccine. This form will securely collect your patient information, and you have the option to register your entire household to save time. You will also be asked vaccination Phase eligibility questions. At the time of vaccination, you may be required to bring proof of employment at a medical facility, etc.',
+    introStatement: 'You are about to register as a patient for the COVID-19 vaccine. This form will securely collect your patient information. At the time of vaccination, you may be required to bring proof of employment at a medical facility, etc.',
```

Comment:
potential reword? (the proof of employment is a bit awkward and doesn't seem likely to be relevant at this time point)
suggested:
'You are about to register as a patient for the COVID-19 vaccine. This form will securely collect your patient information. At the time of vaccination, you may be required to bring proof of identity or proof of eligibility if relevant.',